render/filters.go

```diff
@@ -58,12 +58,26 @@ func ColorConnected(r Renderer) Renderer {
 // Filter removes nodes from a view based on a predicate.
 type Filter struct {
 	Renderer
-	FilterFunc func(report.Node) bool
+	FilterFunc     func(report.Node) bool
+	ReportFiltered bool // false means we don't report stats for how many are filtered
```

Comment:
From an implementation perspective sure, but from a user perspective, I think of the steps as:
1) Nodes are aggregated
2) Nodes are filtered
3) Pseudo-nodes are added

therefore, I wouldn't expect the filter count to include them, because they're an artefact of the view, not of the data.